build.sbt

```diff
@@ -1602,20 +1644,28 @@ lazy val `engine-runner-native` = project
         initializeAtRuntime = Seq(
           // Note [WSLoggerManager Shutdown Hook]
           "org.enso.loggingservice.WSLoggerManager$",
-          "io.methvin.watchservice.jna.CarbonAPI"
+          "io.methvin.watchservice.jna.CarbonAPI",
         )
       )
+      .dependsOn(installNativeImage)
```

Comment:
@hubertp Multiple tasks now depend on `installNativeImage` task. `installNativeImage` should be *synchronized*, i.e., should be called at most once from just one thread. Is there any way how to achieve that in `sbt`? A quick workaround could be to introduce a mutex, but that seems ugly.

Reply:
Added a commit that solves this, I think

Reply:
Thanks, I will do the same for `installGuJS`